vhd/packer/sysprep.ps1

```diff
@@ -0,0 +1,38 @@
+# Stop and remove Azure Agents to enable use in Azure Stack
+# If deployed in azure vm the agents will be added to the vms at deployment time
+Stop-Service WindowsAzureGuestAgent
+Stop-Service WindowsAzureNetAgentSvc
+Stop-Service RdAgent
+# Stop-Service WindowsAzureTelemetryService
```

Comment:
Why not stop and delete WindowsAzureTelemetryService?

Reply:
Moved this to https://github.com/Azure/aks-engine/pull/3977 and answered there